ui/src/components/queries/ModelListModalQuery.js

```diff
@@ -1,117 +1,125 @@
 import React from 'react';
-// import { get } from 'lodash';
+import { get } from 'lodash';
 
 import { useDataContext } from '@overture-stack/arranger-components/dist/DataContext';
-import globals from 'utils/globals';
-import Component from 'react-component-component';
 
-// const fetchData = async ({ setState, modelIds }) => {
-//   const { data } = await api({
-//     endpoint: `/graphql/ModelDataQuery`,
-//     body: {
+import Component from 'react-component-component';
 
-const query = (count) => `query ModelListModal($filters: JSON) {
-            models {
-              extended
-              hits(first: ${count} filters: $filters) {
-                edges {
-                  node {
-                    id
-                    expanded
-                    proteomics_url
-                    source_model_url
-                    source_sequence_url
-                    somatic_maf_url
-                    name
-                    type
-                    split_ratio
-                    growth_rate
-                    primary_site
-                    neoadjuvant_therapy
-                    tnm_stage
-                    molecular_characterizations
-                    age_at_sample_acquisition
-                    vital_status
-                    gender
-                    race
-                    chemotherapeutic_drugs
-                    age_at_sample_acquisition
-                    disease_status
-                    files {
-                      hits{
-                        edges {
-                          node {
-                            file_id
-                            file_url
-                            file_name
-                            file_type
-                          }
-                        }
-                      }
-                    }
-                    therapy
-                    licensing_required
-                    date_of_availability
-                    clinical_diagnosis {
-                      clinical_tumor_diagnosis
-                      site_of_sample_acquisition
-                      histological_type
-                      tumor_histological_grade
-                      clinical_stage_grouping
-                    }
-                    matched_models {
-                      hits{
-                        edges {
-                          node {
-                            name
-                            tissue_type
-                          }
-                        }
-                      }
-                    }
-                  }
+const modelListQuery = `query ModelListModal ($first: Int, $sqon: JSON) {
+    model {
+      hits(first: $first, filters: $sqon) {
+      total
+      edges {
+        node {
+          expanded
```

Comment:
Out of curiosity: is there any reason that there are some new fields in the new query (like `distributor_part_number`), and some old ones that are not in the new query (like `id`)?

Reply:
Update: it looks like you may have accidentally copied the query from `ModelQuery` into this -- at least, looking at the [original/develop](https://github.com/nci-hcmi-catalog/portal/blob/develop/ui/src/components/queries/ModelQuery.js) version of `ModelQuery` it appears to contain all of the new fields that are now present in this query. I'm not sure if we need all of those fields for this one. And I'm unsure if the `id` is necessary, but that is missing.

For quick reference, here's the original `ModelListModalQuery.js`: https://github.com/nci-hcmi-catalog/portal/blob/develop/ui/src/components/queries/ModelListModalQuery.js

Reply:
So some of the 'new' fields do actually appear to be name changes to existing fields, and I'm using definitions found in the GraphQL sandbox.

Let me review what's going on with the ModelListQuery and update.

Reply:
Updated modelList query, thanks for catching